<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstallMainWindow</class>
 <widget class="QMainWindow" name="InstallMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>InstallMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="InstallTabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>768</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Master">
     <attribute name="title">
      <string>MASTER</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Exist MSW List</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>30</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Available MSW packages</string>
      </property>
     </widget>
     <widget class="QListWidget" name="ExistListWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>401</width>
        <height>641</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="AvaliableListWidget">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>80</y>
        <width>531</width>
        <height>641</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>550</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="PTS">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <attribute name="title">
      <string>PTS</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Service">
     <attribute name="title">
      <string>SERVICE</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Slavetab">
     <attribute name="title">
      <string>Slave</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="Settings">
     <attribute name="title">
      <string>SETTINGS</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
